docs/basics/vue.md

```diff
@@ -269,6 +269,8 @@ import { useQuery } from '@urql/vue';
 export default {
   props: ['from', 'limit'],
   setup({ from, limit }) {
+    const shouldPause = computed(() => from === undefined || from === null ||
+                                      limit === undefined || limit === null);
```

Comment:
```suggestion
    const shouldPause = computed(() => from == null || limit == null);
```

Is a bit more terse 😅

Reply:
I admit 😂
Sacrificing the edge case for a more terse code sounds good to me

Reply:
Actually your suggestion only applied to the basics/vue.md example. In basics/react-preact.md there's a similar example but it still has my initial suggestion. Should I open a new PR to fix that?